code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -83,6 +83,22 @@
 /obj/item/melee/baton/get_cell()
 	return cell
 
+/obj/item/melee/baton/mob_can_equip(mob/user, slot, disable_warning = 1)
```

Comment:
Why is the `disable_warning` set to default to `1` (`TRUE`) here? Since the parent has `0` (AKA `FALSE`)

Reply:
Iirc it's because otherwise you get the default warning sent to chat several times because it checks each slot, also without it you get the default cannot store message as well as the one sent when if the baton is on. I'll double check when I get home from work but I believe it was to prevent double warnings.